TombEngine/Scripting/Internal/TEN/Logic/LogicHandler.cpp

```diff
@@ -634,9 +642,9 @@ void LogicHandler::GetVariables(std::vector<SavedVar>& vars)
 
 				case sol::type::userdata:
 				{
```

Comment:
I'm not sure, but shouldn't be here another if case for the vec2 float?

Reply:
Thanks, this might be why it wasn't loading.